Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.h

```diff
@@ -213,11 +213,14 @@ enum
 class I2CSlave
 {
 public:
-  virtual int BusRead(u8 addr, int count, u8* data_out) = 0;
-  virtual int BusWrite(u8 addr, int count, const u8* data_in) = 0;
+  // Kill MSVC warning:
```

Comment:
I don't think that comment is necessary. The dtor could be introduced in the original commit though.